src/styles.css

```diff
@@ -132,3 +143,72 @@ h3 {
   }
 }
 
+#clickedRecipe {
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  right: 0;
+  width: 98%;
+  border-radius: 50px;
+  background-color: darkgray;
+}
+
+#recipeLeftSide {
```

Comment:
I see that you work with ID's. 
Could I know your reason to use them over classes?

I personally, use classes over ID's because it's more applicable in styling (can target multiple items) and if, at a later point you had to override a styling by class, you could do so by adding an ID (which is more specific so would take preference in styling). 
But if you start with ID styling, overriding a style could be a challenge later on.

Totally up to you to make an educated decision here. Just wanted to share my preference and reasoning.

Reply:
Yeah there's definitely some room for drying up some of this css with classes.  I think I opted for IDs because I knew they were all going to be doing pretty different things.  I'll see what I can class out right now.